src/lib/select/select.html

```diff
@@ -7,7 +7,7 @@
   <div class="mat-select-value" [ngSwitch]="empty">
     <span class="mat-select-placeholder" *ngSwitchCase="true">{{placeholder || '\u00A0'}}</span>
     <span class="mat-select-value-text" *ngSwitchCase="false" [ngSwitch]="!!customTrigger">
-      <span *ngSwitchDefault>{{triggerValue}}</span>
+      <span *ngSwitchDefault>{{triggerValue || '\u00A0'}}</span>
```

Comment:
Why the escape sequence instead of a space `' '`?

Reply:
We had a reason for doing it a bit above in the select as well, but I don't remember exactly what it was. I _think_ it's because the browser would still collapse the regular whitespace string.

Reply:
yeah that's the js equivalent of `&nbsp;`